tensorboard/uploader/exporter.py

```diff
@@ -50,9 +51,18 @@
 _FILENAME_METADATA = "metadata.json"
 # Output filename for scalar data within an experiment directory.
 _FILENAME_SCALARS = "scalars.json"
+# Output filename for blob sequences data within an experiment directory.
+# This file does not contain the actual contents of the blobs. Instead,
+# it holds `blob_file_path`s that point to the binary files that contain
+# the blobs' contents, in addition to other metadata such as run and tag names.
+_FILENAME_BLOB_SEQUENCES = "blob_sequences.json"
 
 logger = tb_logging.get_logger()
 
+_BLOBS_DIR = "blobs"
+_BLOB_FILE_PREFIX = "blob_"
+_BLOB_FILE_SUFFIX = ".bin"
```

Comment:
nit: Please collocate these constants with their friends rather than
being sundered by `logger = tb_logging.get_logger()`. (Also, imho it’s
clearer to have `_FILENAME_BLOBS_DIR`, `_FILENAME_BLOB_PREFIX`, etc.,
but if you really want the mixed-and-matched names I can live with it.)